### ISSUE TITLE
Why do you disable decay weight for bias?

### ISSUE THREAD

**Issue by acgtyrant** [chenyuntc/simple-faster-rcnn-pytorch#18]
See https://github.com/chenyuntc/simple-faster-rcnn-pytorch/blob/master/model/faster_rcnn.py#L273

The Fast RCNN paper says:

> A momentum of 0:9 and parameter decay of 0:0005 (on weights and biases) are used.

The Faster RCNN paper says:

> We use a momentum of 0.9 and a weight decay of 0.0005 [37].

And the origin code dose not set the special weight decay [in the prototxt](https://github.com/rbgirshick/py-faster-rcnn/blob/master/models/pascal_voc/VGG16/faster_rcnn_end2end/train.prototxt).

**Comment by chenyuntc** [comment]
Yes, You are right. 

You may  refer to from https://arxiv.org/pdf/1702.02138.pdf

>Other minor changes that does not seem to affect the performance include: 1) double the learning rate for bias; 2) stop weight decay on bias; 3) remove aspect-ratio group- ing (introduced to save memory); 4) exclude ground-truth bounding boxes in the RoIs during training, since they are not accessible during testing and can bias the input distribu- tion for region classification.
